datadog-monitors-monitor-handler/src/datadog_monitors_monitor/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "2.0.0"
```

Comment:
where is this used? 
Ahh is it for the telemetry?

Reply:
Yes telemetry

Reply:
```suggestion
__version__ = "2.2.0"
```